releases/protocol/release-specs-v1.7.14-Interim.md

```diff
@@ -0,0 +1,124 @@
+[comment]: <> (tags/v1.7.16)
+
+# Contents
+
+This document explains the contents of the rc/v1.7.next1 release codenamed Interim. It is split in 2 sections:
```

Comment:
Codename should be a star name.

Reply:
Decided with @schimih that this one would be called Interim for now.

Reply:
https://dictionary.cambridge.org/dictionary/english/interim